cfeScraping/index.js

```diff
@@ -0,0 +1,25 @@
+const puppeteer = require('puppeteer');
+const URL = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+OpenBrowser = async () => {
```

Comment:
Pls define the scope of the function, var, let, const. This is really bad practice